### ISSUE TITLE
Simple Calculator tutorials should include a note to enable unsafe code in project properties

### ISSUE THREAD

**Issue by trydalch** [unoplatform/uno#18510]
### On which page?

https://platform.uno/docs/articles/external/workshops/simple-calc/modules/MVUX-XAML/03-Creating%20the%20Layout/README.html

### What's wrong?

SimpleCalc no longer builds after step 3 (Creating the Layout) unless you Allow Unsafe Blocks in your project properties.

It likely needs to be updated on all 4 walk throughs (MVUX, MVVM, XAML, C#)

**Comment by AndrewKeepCoding** [comment]
Seems to be a duplicate of #18464.
